src/main/java/racingcar/CarRace.java

```diff
@@ -46,7 +46,32 @@ public int getTryCnt() {
         return raceResult;
     }
 
+    public List<Car> getWinnerCars() {
+        List<Car> winnerCar = new ArrayList<>();
+        int winnerDistance = getWinnerDistance(raceResult);
+        RaceSnapshot lastRaceSnapShot = raceResult.get(raceResult.size()-1);
+        for(Car car : lastRaceSnapShot.cars){
+            if(car.getDistance() == winnerDistance){
```

Comment:
getter 사용은 객체의 캡슐화를 위해 지양해야 합니다 😢 
getter 를 사용하기 보다는 객체에 메세지를 보내보면 어떨까요? 😄 
```suggestion
            if(car.equalDistance(winnerDistance)){
```